MediaGalleryUi/view/adminhtml/web/js/grid/sortBy.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'Magento_Ui/js/grid/sortBy'
+], function (Element) {
+    'use strict';
+
+    return Element.extend({
+        defaults: {
+            columnIndexMap: {},
+        },
+
+        /**
+         * Prepared sort order options
+         */
+        preparedOptions: function (columns) {
+            var index = 0;
+
+            if (columns && columns.length > 0) {
+                columns.map(function (column, i) {
+                    if (column.sortable === true) {
+                        this.options.push({
```

Comment:
Is it possible to provide these options from XML configuration of the sorting component?